### ISSUE TITLE
Another error

### ISSUE THREAD

**Issue by lcnvdl** [enigma-dev/lgmplugin#30]
Operating System: Windows 7
Version: 6.1
Architecture: amd64

Java Vendor: Oracle Corporation
Version: 1.7.0_51

Available processors (cores): 2
Free memory (bytes): 47332320
Maximum memory (bytes): 924844032
Total memory available to JVM (bytes): 79167488

File system root: C:\
Total space (bytes): 125724258304
Free space (bytes): 25067986944
Usable space (bytes): 25067986944

File system root: D:\
Total space (bytes): 194241359872
Free space (bytes): 144842629120
Usable space (bytes): 144842629120

File system root: E:\
Total space (bytes): 0
Free space (bytes): 0
Usable space (bytes): 0

File system root: F:\
Total space (bytes): 0
Free space (bytes): 0
Usable space (bytes): 0

File system root: Z:\
Total space (bytes): 249995624448
Free space (bytes): 30838194176
Usable space (bytes): 30838194176

Stack trace:

java.lang.UnsatisfiedLinkError: Unable to load library 'compileEGMf': Native library (win32-x86-64/compileEGMf.dll) not found in resource path ([file:/C:/Users/Jorge/Downloads/Enigma/ENIGMA/enigma-dev/plugins/enigma.jar])
	at com.sun.jna.NativeLibrary.loadLibrary(NativeLibrary.java:271)
	at com.sun.jna.NativeLibrary.getInstance(NativeLibrary.java:398)
	at com.sun.jna.Library$Handler.<init>(Library.java:147)
	at com.sun.jna.Native.loadLibrary(Native.java:412)
	at com.sun.jna.Native.loadLibrary(Native.java:391)
	at org.enigma.EnigmaRunner.attemptLib(EnigmaRunner.java:247)
	at org.enigma.EnigmaRunner.access$1(EnigmaRunner.java:240)
	at org.enigma.EnigmaRunner$3.run(EnigmaRunner.java:178)

**Comment by RobertBColton** [comment]
@lcnvdl I believe this is the result of you using a 64 bit Java installation. Can you please tell me what java -version prints from command prompt? ENIGMA currently only supports 32 bit on Windows because it does not have a 64 bit version of MinGW and JNA in the plugin must use the architecture. LateralGM will work fine by itself with 64 bit, but if you intend to use the ENIGMA plugin you will need to install a 32 bit Java version and run them in parallel. We also no longer support Java 6, the minimum supported Java version is 1.7 for the newer versions of LateralGM.
Please see the wiki for more details:
http://enigma-dev.org/docs/Wiki/Troubleshoot#Why_am_I_getting_an_error_about_compileEGMf.3F
http://enigma-dev.org/docs/Wiki/Install:Parallel_Java
Let me know if that fixes everything.

**Comment by RobertBColton** [comment]
Closing as duplicate of #20
